dts/bindings/flash_controller/nordic,nrf53-flash-controller.yaml

```diff
@@ -0,0 +1,8 @@
+title: Nordic NVMC
+
+description: >
+    This binding gives a base representation of the Nordic NVMC
```

Comment:
Could give a more meaningful description. Not sure what "base representation" was ever supposed to mean.

Something like

```
description: |
    Nordic NVMC (<explain acronym>). See datasheet at ...
```

Or whatever other information might be useful.